internal/driver/commands.go

```diff
@@ -246,6 +246,22 @@ func helpText(s ...string) string {
 	return strings.Join(s, "\n") + "\n"
 }
 
+func addVariablesPostFix(name string, vr *variable) string {
```

Comment:
Nit: Perhaps rename to `variablePostFix`? 
When I read the current name I think it might imply that this function would return something like "name=<f>".

Reply:
Done.